src/main/java/pl/javastart/task/Tasks.java

```diff
@@ -0,0 +1,54 @@
+package pl.javastart.task;
+
+public class Tasks {
+    private final String taskName;
+    private final String taskDescription;
+    private int priority;
+    Person person;
+
+    public Tasks(String taskName, String taskDescription, int priority, Person person) {
+        this.taskName = taskName;
+        this.taskDescription = taskDescription;
+        this.priority = priority;
+        this.person = person;
+    }
+
+    public Tasks(String taskName, String taskDescription, int priority) {
+        this.taskName = taskName;
+        this.taskDescription = taskDescription;
+        this.priority = priority;
+    }
+
+    public Tasks(String taskName, String taskDescription) {
+        this.taskName = taskName;
+        this.taskDescription = taskDescription;
+    }
```

Comment:
W ramach jednego konstruktora możesz wywoływać też inne konstruktory z tej samej klasy przez this(), np. `this(name, description)` co pozwala uniknąć duplikowania kodu